Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -1374,6 +1377,15 @@ def audit_urls
       next unless u =~ %r{https?://(?:central|repo\d+)\.maven\.org/maven2/(.+)$}
       problem "#{u} should be `https://search.maven.org/remotecontent?filepath=#{$1}`"
     end
+
+    return unless @online
+    urls.each do |u|
+      begin
+        nostdout { curl "--connect-timeout", "15", "-o", "/dev/null", "-r", "0-0", u }
```

Comment:
Good idea with the arguments!

From what I've read, a `HEAD` request would be preferable but some mirrors seem to block `HEAD` requests (one example being github releases e.g. https://github.com/admesh/admesh/releases/download/v0.98.2/admesh-0.98.2.tar.gz). I'm hoping that requesting the first byte will be more reliable.